css-borders-4/Overview.bs

```diff
@@ -372,23 +929,112 @@ The 'border-top-radius', 'border-right-radius',
+	border-radius-001.xht
+	border-radius-002.xht
+	border-radius-003.xht
+	border-radius-004.xht
+	border-radius-005.xht
+	border-radius-006.xht
+	border-radius-007.xht
+	border-radius-008.xht
+	border-radius-009.xht
+	border-radius-010.xht
+	border-radius-011.xht
+	border-radius-012.html
+	border-radius-013.html
+	border-radius-clip-001.html
+	border-radius-clip-002.htm
+	border-radius-clipping-002.html
+	border-radius-clipping-with-transform-001.html
+	border-radius-css-text.html
+	border-radius-dynamic-from-no-radius.html
+	border-radius-horizontal-value-is-zero.html
+	border-radius-shorthand-002.html
+	border-top-left-radius-001.xht
+	border-top-left-radius-004.xht
+	border-top-left-radius-005.xht
+	border-top-left-radius-010.xht
+	border-top-left-radius-011.xht
+	border-top-left-radius-014.xht
+	border-top-right-radius-001.xht
+	border-top-right-radius-004.xht
+	border-top-right-radius-005.xht
+	border-top-right-radius-010.xht
+	border-top-right-radius-011.xht
+	border-top-right-radius-014.xht
+	box-shadow-border-radius-001.html
+	css-border-radius-001.html
+	inheritance.sub.html
+	inner-border-non-renderable.html
+	parsing/border-radius-computed.html
+	parsing/border-radius-invalid.html
+	parsing/border-radius-valid.html
+	parsing/webkit-border-radius-valid.html
+</wpt>
 
 <h4 id="corner-sizing-shorthand">
 Sizing All Corners At Once:
 The 'border-radius' shorthand</h4>
 
-	<pre class="propdef">
+	<pre class="propdef shorthand">
 		Name: border-radius
 		Value: <<length-percentage [0,&infin;]>>{1,4} [ / <<length-percentage [0,&infin;]>>{1,4} ]?
-		Initial: 0
-		Applies to: all elements, except table element when 'border-collapse' is ''collapse''
-		Inherited: no
-		Animation type: see individual properties
 	</pre>
 
-	See [[CSS3BG]].
+	The 'border-radius' shorthand sets all four 'border-*-radius' properties.
+	If values are given before and after the slash,
+	then the values before the slash set the horizontal radii
+	and the values after the slash set the vertical radii.
+	If there is no slash, then the values set the radii in both axes equally.
+	The four values for each radii are given in the order
+	top-left, top-right, bottom-right, bottom-left.
+	If bottom-left is omitted it is the same as top-right.
+	If bottom-right is omitted it is the same as top-left.
+	If top-right is omitted it is the same as top-left.
 
-<h3 id=corner-shaping>
+	<div class="example">
+		<pre>
+		border-radius: 4em;
+		</pre>
+		is equivalent to
+		<pre>
+		border-top-left-radius:     4em;
+		border-top-right-radius:    4em;
+		border-bottom-right-radius: 4em;
+		border-bottom-left-radius:  4em;
+		</pre>
+		and
+		<pre>
+		border-radius: 2em 1em 4em / 0.5em 3em;
+		</pre>
+		is equivalent to
+		<pre>
+		border-top-left-radius:     2em 0.5em;
+		border-top-right-radius:    1em 3em;
+		border-bottom-right-radius: 4em 0.5em;
+		border-bottom-left-radius:  1em 3em;
+		</pre>
+	</div>
+
+<h3 id="corner-shaping">
 Corner Shaping: the 'corner-*-shape' properties</h3>
```

Comment:
I don't think autolink will work here :)

Reply:
Right. I've added the it together with similar terms to the ignored terms.